js/src/components/paid-ads/ads-campaign.js

```diff
@@ -26,18 +28,33 @@ import EditPaidAdsCampaignFormContent from '.~/components/paid-ads/edit-paid-ads
  * Please note that this component relies on an AdaptiveForm's context, so it expects
  * a context provider component (`AdaptiveForm`) to existing in its parents.
  *
- * @fires gla_documentation_link_click with `{ context: 'create-ads', link_id: 'see-what-ads-look-like', href: 'https://support.google.com/google-ads/answer/6275294' }`
- * @fires gla_documentation_link_click with `{ context: 'edit-ads', link_id: 'see-what-ads-look-like', href: 'https://support.google.com/google-ads/answer/6275294' }`
+ * @fires gla_documentation_link_click with `{ context: 'create-ads' | 'edit-ads' | 'setup-ads', link_id: 'see-what-ads-look-like', href: 'https://support.google.com/google-ads/answer/6275294' }`
  *
  * @param {Object} props React props.
  * @param {Campaign} [props.campaign] Campaign data to be edited. If not provided, this component will show campaign creation UI.
  * @param {() => void} props.onContinue Callback called once continue button is clicked.
+ * @param {'create-ads'|'edit-ads'|'setup-ads'} props.trackingContext A context indicating which page this component is used on. This will be the value of `context` in the track event properties.
```

Comment:
💅 

An alternative solution for knowing the context could be to use the paths?

Reply:
Ideally, the `AdsCampaign` component is as agnostic as possible about which pages it would be used. So using the `trackingContext` prop given by the consumer sides of this component would be closer to the purpose.